pyathena/connection.py

```diff
@@ -66,6 +66,7 @@ def __init__(
         retry_config: Optional[RetryConfig] = None,
         cursor_class: Type[BaseCursor] = Cursor,
         kill_on_interrupt: bool = True,
+        session: Any = None,
```

Comment:
This could use `Optional[Session]` rather than `Any`

Reply:
Fixed: https://github.com/laughingman7743/PyAthena/pull/252/commits/8603a2d97b80a75bc48ea463ace3b18efef6a989